axolotl-web/src/assets/style.scss

```diff
@@ -44,6 +43,9 @@ $link-color: $primary;
 .receipt .message {
   background-color: $recieved-message-bg;
```

Comment:
with https://github.com/nanu-c/axolotl/commit/0727b7a76b2f9d19d05b6d97d4b1ea2932df935e .receipt is always not there.

Reply:
I'll do a front-end cleaning... later :stuck_out_tongue: